specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -37,10 +37,10 @@ export function StatsPage(): JSX.Element {
   const backEndResult = useBackendApi();
 
   return (
-    <div className="mx-auto mt-0 flex h-full flex-col gap-4 bg-gray-200 p-4">
+    <div className="flex h-full flex-col gap-4 overflow-y-auto bg-[color:var(--form-background)] p-4">
```

Comment:
Can you use one of these container here? https://github.com/specify/specify7/blob/4963b6f2d0700d5f95287a1c01a1de33f74bb1e0/specifyweb/frontend/js_src/lib/components/Atoms/index.tsx#L20-L39

If not, can you at least use one of these class names? https://github.com/specify/specify7/blob/4963b6f2d0700d5f95287a1c01a1de33f74bb1e0/specifyweb/frontend/js_src/lib/components/Atoms/className.ts#L65-L70